x-pack/plugins/canvas/public/expression_types/arg_types/container_style/index.js

```diff
@@ -24,11 +25,19 @@ const wrap = Component =>
     },
   })(Component);
 
-export const containerStyle = () => ({
+const containerStyleUI = intl => ({
   name: 'containerStyle',
-  displayName: 'Container Style',
-  help: 'Tweak the appearance of the element container',
+  displayName: intl.formatMessage({
+    id: 'xpack.canvas.expressionTypes.containerStyle.containerStyleLabel',
```

Comment:
xpack.canvas.expressionTypes.containerStyle => xpack.canvas.containerStyle for all ids in this file